tests/ci/config.json

```diff
@@ -0,0 +1,6 @@
+{
+    "subscription_id": "<>",
+    "resource_group": "recommender",

```

Comment:
I think it would be useful to populate this either from key-vault entries or variables in the build pipeline, that way we don't need a code change to adjust the parameters.

alternatively if the az ml cli extension is available and allows this to be downloaded programmatically that would be preferable, something like:

```
az ml workspace show -g recommender -w RecoWS -o json > config.json
```

Reply:
Yes, that's on the list of things to do.  The key vault is already set up and I'd like to get rid of the json file entirely.

Reply:
done